CppCoreGuidelines.md

```diff
@@ -1457,7 +1457,7 @@ In the following example, it is not clear from the interface what `time_to_blink
 
     void use()
     {
-        blink_led(1500ms);
+        blink_led(1500ms);  // good -- the unit is explicit (std::chrono_literals: c++14).
```

Comment:
std::literals::chrono_literals (since C++14)
pls refer to https://en.cppreference.com/w/cpp/symbol_index/chrono_literals

Reply:
if it has to say something here (not clear if it does), it could just say "good -- the unit is explicit"

Reply:
> std::literals::chrono_literals (since C++14)
> pls refer to https://en.cppreference.com/w/cpp/symbol_index/chrono_literals

Thx, I updated it.

Reply:
> if it has to say something here (not clear if it does), it could just say "good -- the unit is explicit"

I wanted to highlight that the expression "1500ms" is valid since C+14 because there was a line mentions C++11 just above to remove confusion.

Reply:
@markyin
> `std::literals::chrono_literals` and `std::chrono_literals` are both correct.

I agree with Sergey that there's not really any need to say this. I'd remove the "(C++11)" bit above (and fix the broken English in that sentence above). We don't need to introduce C++11 types as though they're novel, they've been around for more than a decade.